contents/docs/integrate/server/python.mdx

```diff
@@ -175,9 +175,12 @@ When using them with one of libraries, you should check if a feature flag is ena
 
 **How to check if a flag is enabled**
 
+> **Note:** Whenever we face an error computing the flag, the library returns `None`, instead of `true` or `false` or a string variant value.
```

Comment:
```suggestion
> **Note:** Whenever we face an error computing the flag, the library returns `None`, instead of `true`, `false`, or a string variant value.
```